tests-manual/debugger/README.md

```diff
@@ -0,0 +1,45 @@
+Debugger Command Line Test Programs
+===================================
+
+This directory contains a set of utilities used to manually test the pipeline debugger features. These tests should be run against a live system with the debugger facilities built into it.
+
+To start testing;
+
+  - Change directory to this directory
+
+  - Type the command
+
+    .. code-block:: console
+
+       ./debug <ServiceName>
+
+    Where *<ServiceName>* is the name of a south or north service. You will need to quote the name if it contains whitespace or wildcard characters that have meaning to the shell.
+
+   - Attach the debugger to the pipeline by usign the attach command
+
+    .. code-block:: console
+
+       attach
+
+The debugger is now attached to the pipeline and collecting one reading at each point in the pipeline.
+
+To see the list of command available type the commands command
```

Comment:
"_To see the list of command available type the commands command_"

- Instead, "To see the list of available commands...?"
- Also perhaps italicize "_commands_" because "commands command" looks odd.

Reply:
Not changed?